ListOfAllAccountControler.cls

```diff
@@ -0,0 +1,180 @@
+    public String accName {get; set;}
+    // Declare the Integer with getters and setters Which saves the size of the account list.
+    public Integer accountSize{get;set;}
+    // Declare the Integer with getters and setters Which saves the size of the contact list.
+    public Integer contactSize{get;set;}
+    // Declare the String with getters and setters to save the String that helps to search.
+    public String searchStringAccount{get;set;}
+    // Declare the list of string used for creating the adding A to Z Variables.
+    public List<String> listAToZInstance;
+    // Initialize the Flag to false.
+    public Boolean flagCheck=false;
+    // Declare the String with getters and Setters Which saves the string to search the element. 
+    public String alphaName {get; set;}
+      // Declare the Id of the getters and setters Which saves the Id Instance of Contact.
+    public Id idcontact{get; set;}
+ 
+   /**
+    * This is a Getter that will fetch all the records of the Account when the flag is false 
+    * Or if the Flag is true fetch the searched account.
+    */
+    public List<Account> getAccount()
+    { 
+        // Check if the Flag is false.
+        if(flagCheck==false)
+        {
+            // Fetch all the Accounts record.
+             account=[Select a.Website, a.Phone, a.Name, a.Email_ID__c,a.Id 
+                      From Account a];
+             // Save total size of the Account List.
+             accountSize=account.size();
+        }
+        else 
+        {   
+            // Fetch the Accounts record that are searched.
+              account=[Select a.Website, a.Phone, a.Name, a.Email_ID__c,a.Id 
+                             From Account a
+                             Where Account.name like :'%'+ searchStringAccount.trim() +'%'];
+              // Save total size of the Account List.
+              accountSize=account.size();
+        }
+        // Return the Account list.
+        return account;
+    }
+ 
+   /**
+    * This is a Getter that will fetch all the records of the Contact related to that account 
+    * when the flag is false Or if the Flag is true fetch the searched account.
+    */
+    public List<Contact> getcontactList()
+    {
+        // Get the id passed from the accounts page.
+        idAccount = ApexPages.currentPage().getParameters().get('aid');
+        // Check for the Flag.
+        if(flagCheck==false)
+        {
+            // Get the name of that account.
+            accName = ApexPages.currentPage().getParameters().get('aname');
+            // Fetch the contact records that are related to that account.
+            contactList=[Select Id,Name 
+                         from Contact 
+                         where Account.id =: idAccount];
+            // Saves the size of the list.
+            contactSize=contactList.size();
+            // Return the contact list 
+                  return contactList;
```

Comment:
check indentation in complete class and for spacing , variable names and other best practices and change it